docs/fundamentals/layouts/grid.md

```diff
@@ -41,3 +40,193 @@ content to accommodate the browser window's width, regardless of how wide it
 becomes.
 
 ![Layout responsive](images/layout-fluid-fixed.png)
+
+## Code ---
+
+Leverage our mobile-first flexbox grid to create layouts of any complexity using a 12-column system, six responsive [breakpoints](./breakpoints.md), and a wide range of predefined utility classes.
```

Comment:
@spliffone  We already give credit to Bootstrap in the README but the grid system is probably one of the core concepts from Bootstrap that we should also give credit here directly - and it also helps people to understand the concept.

I don't think we need to repeat this for everything we derrive from Bootstrap, but here it just makes sense IMHO.

```suggestion
The flexbox grid is used to create layouts of any complexity using a 12-column system, six responsive [breakpoints](./breakpoints.md), and a wide range of predefined utility classes. The grid system is based on the well-known and beloved Bootstrap grid.
```